doc/nrf/app_dev/device_guides/nrf54l/building_nrf54l.rst

```diff
@@ -0,0 +1,108 @@
+.. _building_nrf54l:
+
+Building and programming with nRF54L15 DK
+#########################################
+
+This guide provides instructions on how to build and program the nRF54L15 development kit.
+Whether you are working with single or multi-image builds, the following sections will guide you through the necessary steps.
+
+Depending on the sample, you must program only the application core or both the VPR and the application core.
+Additionally, the process will differ based on whether you are working with a single-image or multi-image build.
+
+Using the command line
+======================
+
+To build samples supporting the nRF54L15 DK from the command line, use :ref:`west <zephyr:west>`.
+For programming you can choose either west, incorporating nrfjprog that from the `nRF Command Line Tools`_, or :ref:`nRF Util <toolchain_management_tools>`.
+
+.. tabs::
+
+   .. group-tab:: Separate images
+
+      To build and program the application sample and the VPR sample as separate images, follow the instructions in :ref:`programming_cmd` for each of the samples.
+
+      See the following instructions for programming the images separately:
+
+      .. tabs::
+
+         .. group-tab:: west
+
+            1. |open_terminal_window_with_environment|
+            #. Run the following command to erase the flash memory of the VPR core and program the VPR sample:
+
+               .. code-block:: console
+
+                  west flash --erase
+
+            #. Navigate to the build folder of the application sample and run the same command to erase the flash memory of the application core and program the application sample:
+
+               .. code-block:: console
+
+                  west flash --erase
```

Comment:
And this `--erase` option seems to have the same effect - removing the code programmed in the previous step.